Sources/BraintreePayPalMessaging/BTPayPalMessagingLogoType.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PayPalMessages
+
+/// Logo type option for a PayPal Message
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingLogoType {
+
+    /// PayPal logo positioned inline within the message
+    case inline
+
+    /// Primary logo including both the PayPal monogram and logo
+    case primary
+
+    /// Alternative logo including just the PayPal monogram
+    case alternative
+
+    /// "PayPal" as bold text inline with the message
+    case none
```

Comment:
instead of `none`, are we ok with the term `default`,  or `minimal`? "None" seems misleading in that there _is_ going to be a logo, it's just really simple.

Maybe this is something we check in with Product for their thoughts

Reply:
`default` is a protected case name in Swift for enums. I am good with `minimal`, `simple`, or `basic`. Do you have a preference?

Reply:
I like `simple`! Let's go with that

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1153/commits/afb0e5af39ad353985e8807618d43ee56b4a95a5